modules/js/rotate.js

```diff
@@ -2,123 +2,158 @@
 rotateModule = true;
 spurMod = true;
 planetaryMod = false;
-var gear1Radius = 80
-var gear2Radius = 80
 // change gear body when changing size when in rotating motion
-function changeBodyRotate(index){
-  for(var i=0; i<1;i++){
+function changeBodyRotate(index) {
     // remove ui motor sprite body
-    if(compositeArray[index].bodies[1]){
+    if (compositeArray[index].bodies[1]) {
       Composite.remove(compositeArray[index], compositeArray[index].bodies[1]);
     }
     // remove gear body
     Composite.remove(compositeArray[index], compositeArray[index].bodies[0]);
     // store constraint position values
-    var tmpConstraintXPoint
-    if(index == 0){
-      tmpConstraintXPoint = (window.innerWidth)*(0.75*0.45)-(radius+(toothHeight*0.6))
+    let tmpConstraintXPoint;
+    if (index == 0) {
+      tmpConstraintXPoint =
+        window.innerWidth * (0.75 * 0.45) - (radius + toothHeight * 0.6);
+    } else if (index == 1) {
```

Comment:
Consider ===.  I won't comment on all of these equalities, but consider strict equality as part of moving to ES6:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality

Reply:
My bad, I'll go back through and change the operators